sdks/python/apache_beam/runners/direct/sdf_direct_runner_test.py

```diff
@@ -157,6 +152,9 @@ def run_sdf_read_pipeline(
 
       assert_that(pc1, equal_to(expected_data))
 
+      # TODO(chamikara: verify the number of times process method was invoked
+      # using a side output once SDFs supports producing side outputs.
```

Comment:
Btw what will currently happen if an SDF attempts to produce side outputs? I realized that Python doesn't have a ProcessContext like Java does, so there's no explicit code that fails if it tries to.

Reply:
We raise an error during transform overriding: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/pipeline.py#L201